artificial-scrum-master/apps/artificial.scrum.master.frontend/src/app/features/sprint-stats/components/burndown-chart/burndown-chart.component.html

```diff
@@ -0,0 +1,15 @@
+<div>
+  <p class="mat-subtitle-2 mt-2">{{ 'Stats.BurndownChart' | translate }}</p>
+  <apx-chart
+    [series]="burndownChartOptions().series || []"
```

Comment:
to || to jakieś jsowe sztuczki tak?

Reply:
Jak dasz || to jeśli to po lewej nic nie zwróci to przypisze to po prawej

Reply:
Byłem ciekaw co Łukasz odpisze :|